taxi/models.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=100, unique=True)
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=100)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL,
+                                     related_name="cars")
```

Comment:
The `related_name` for the `drivers` field in the `Car` model is set to 'cars'. According to the checklist, the `related_name` should be the name of the model in plural. In this case, it should be 'drivers'.